webapp/frontend/src/components/AddTodo.js

```diff
@@ -0,0 +1,35 @@
+// TODO: Replace code in this file over to NSMQ AI frontend components
```

Comment:
the sample code uses the React ToDo example. As the webapp work is being done these areas will be removed accordingly.